app/views/idv/in_person/state_id.html.erb

```diff
@@ -6,6 +6,12 @@
   <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.state_id')) %>
 <% end %>
 
+<%= render AlertComponent.new(
```

Comment:
question for my understanding - are we able to know if a user's id is expired or is this an alert that everyone sees when they arrive on this page?

Reply:
@svalexander Great question! This alert is static. It's an attempt to help users understand what kinds of IDs are acceptable, so it should display regardless of the status of state id.